lib/unleash/feature_toggle.rb

```diff
@@ -32,6 +32,10 @@ def is_enabled?(context, default_result)
       result
     end
 
+    def resolve_stickiness
+      self.variant_definitions&.map(&:stickiness)&.compact&.first || "default"
+    end
```

Comment:
Could you double check that this method is in the private section? (Can't see from my mobile).

Reply:
Spot on - it's been moved!